
This patch adds DMA defines needed by imxmmc.c to compile.

Signed-of-by: Julien Boibessot <julien.boibessot@armadeus.com>
---

--- linux-2.6.27-rc9/arch/arm/plat-mxc/include/mach/dma-mx1-mx2.h	2008-10-16 19:42:15.000000000 +0200
+++ linux-2.6.27-rc9.mod/arch/arm/plat-mxc/include/mach/dma-mx1-mx2.h	2008-10-16 18:41:31.000000000 +0200
@@ -44,6 +44,11 @@
 #define IMX_DMA_ERR_BUFFER    (1 << 3)
 #define IMX_DMA_ERR_TIMEOUT   (1 << 4)
 
+#define IMX_DMA_CCR(x)  (0x8c + ((x) << 6))    /* Control Registers */
+
+/* The type to distinguish channel numbers parameter from ordinal int type */
+typedef int imx_dmach_t;
+
 int
 imx_dma_config_channel(int channel, unsigned int config_port,
 	unsigned int config_mem, unsigned int dmareq, int hw_chaining);
